rs/nns/governance/src/neuron/types.rs

```diff
@@ -1693,6 +1731,9 @@ impl NeuronBuilder {
         #[cfg(not(test))]
         let known_neuron_data = None;
 
+        let recent_ballots_next_entry_index =
+            Some(recent_ballots.len() % MAX_NEURON_RECENT_BALLOTS);
```

Comment:
Since you are setting `recent_ballots_next_entry_index` here to a `Some`, without reversing `recent_ballots`, it looks like you expect the caller to put `recent_ballots` in the desired ordering already. Are you sure all the callers have be updated?

Reply:
That's a good point, I changed it to not make that assumption anymore.